db/models.py

```diff
@@ -1 +1,30 @@
 from django.db import models
+
+
+class Race(models.Model):
+    RACE = [("Elf", "Elf"),
+            ("Dwarf", "Dwarf"),
+            ("Human", "Human"),
+            ("Ork", "Ork")]
+    name = models.CharField(max_length=255, unique=True, choices=RACE)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
```

Comment:
The 'bonus' field in the Skill model is defined as a CharField. Consider whether this is the appropriate field type based on the data it will store. If it's meant to store numerical values, a different field type might be more suitable.